src/main/java/use_case/search/SearchByCriteriaOutputData.java

```diff
@@ -0,0 +1,20 @@
+package use_case.search;
+
+/**
+ * Output data class for search results.
+ */
+public class SearchByCriteriaOutputData {
+    private String movie;
```

Comment:
the variable name (and related getter/setter names) could probably be changed to fit what it does better, something like searchResult.